modules/signing/signing.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Tell build process to exit if there are any errors.
+set -oue pipefail
+
+CONTAINER_DIR="/usr/etc/containers"
+MODULE_DIRECTORY="${MODULE_DIRECTORY:-"/tmp/modules"}"
+
+echo "Setting up container signing in policy.json and cosign.yaml for $IMAGE_NAME"
+echo "Registry to write: $IMAGE_REGISTRY"
+
+if ! [ -d "$CONTAINER_DIR"]; then
+    mkdir -p "$CONTAINER_DIR"
+fi
+
+if ! [ -d $CONTAINER_DIR/registries.d]; then
```

Comment:
Should probably add quotes `""`

Reply:
```suggestion
if ! [ -d $CONTAINER_DIR/registries.d ]; then
```
And a space before the closing `]`

Reply:
```suggestion
if ! [ -d "$CONTAINER_DIR/registries.d" ]; then
```

TIL how to make suggestions. Here's one with both the space and quotes.